cypress/tests/ui/flowchart/banners.cy.js

```diff
@@ -1,8 +1,8 @@
 describe('Banners in Kedro-Viz', () => {
-  beforeEach(() => {
-    // Clears localStorage before each test
-    cy.clearLocalStorage();
-  });
+  // beforeEach(() => {
+  //   // Clears localStorage before each test
+  //   cy.clearLocalStorage();
+  // });
```

Comment:
Delete this?

Reply:
so i kept it because I will probably uncomment it and add it back. This PR is actually a temp PR, we will undo all the changes next release. Because we won't need deprecation banner anymore

Reply:
let me remove it anyway. coz in another place, i removed it.